core/driver/sources/src/main/resources/reference.conf

```diff
@@ -0,0 +1,113 @@
+// Default configuration for the org.virtuslab.ideprobe.config.IdeProbeConfig
+// Comments starting with the # sign are for non-default configs which can be used by users.
+// Comments starting with the // signs are for descriptions and guides.
+probe {
+
+  // `intellij` is for org.virtuslab.ideprobe.config.IntellijConfig - default values below:
+  intellij {
+    plugins = []
+    version {
+      build = "212.5080.55"
+      release = "2021.2.1"
+    }
+    // Use below config with proper path if you want to use an existing Intellij instead of downloading a version.
+    # path = ""
+  }
+
+  // `workspace` is for `Option[org.virtuslab.ideprobe.config.WorkspaceConfig]` - default None,
+  // hence comments added as a reference. You can specify the workspace by using the below configurations.
+  # workspace {
+  // First option is to use a Default workspace - by just specifying `probe.workspace.path` with a valid string.
+  // `probe.workspace.path` can point to:
+  // a) a path on local filesystem (file or jar)
+  // or
+  // b) an online resource (http/https) - which can be for example a github repository
+  #   path = ""
+  //
+  // If `probe.workspace.path` is set and points to a git repository - you can choose one of three additional options:
+  // `branch`, `tag` or `commit` - to specify the reference which should be checked out on the workspace setup.
+  // Only one of them can be specified. You can as well not use any of these 3 options - in such scenario the default
+  // branch will be checked out just like with a plain `git clone` command.
+  #   branch = ""
+  #   tag = ""
+  #   commit = ""
+  //
+  // Another option is to use an Existing workspace by passing a proper path to its directory
+  // - use only `existing` config. In this case, `probe.workspace.path` MUST NOT be defined.
+  // `probe.workspace.existing` can be used for scenarios where you want to reuse an existing workspace instead of
+  // copying a workspace each time when tests run. It might be beneficial for big projects / repos.
+  #   existing = ""
+  # }
+
+  // `resolvers` is for `org.virtuslab.ideprobe.config.DependenciesConfig.Resolvers` - default values below
+  resolvers {
+    intellij {
+      repositories = []
+    }
+    plugins { }
```

Comment:
Hmm +1, what's expected here? pls provide some example, this might be useful for getting rid of `ui-test.plugin.path` system property in git-machete-intellij-plugin (that we discussed right now) and replacing it with a proper Typesafe config entry